lms/djangoapps/email_marketing/tasks.py

```diff
@@ -294,6 +296,72 @@ def _get_course_content(course_url, sailthru_client, email_config):
     return response
 
 
+def _get_or_create_user_lists(sailthru_client, site_domain=None):
+    """
+    Get the user list name from cache if exists else create one and return the name
+    :param sailthru_client:
+    :param user_lists:
+    :return: list name if exists or created else return None
+    """
+    email_config = EmailMarketingConfiguration.current()
+    if not email_config.enabled:
+        return
```

Comment:
@schenedx We already check for an enabled EmailMarketingConfiguration in `update_user`, no need to check it again here. Perhaps we could add a comment to the docstring which indicates that callers of this function should perform the enabled check.